cmd/agent-operator/agent-example-config.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     app: grafana-agent-example
 spec:
-  image: grafana/agent:v0.18.2
+  image: grafana/agent:latest
```

Comment:
I needed to temporarily make this change since v0.18.2 doesn't have any of the logs changes needed for the operator to work & pull logs. Maybe we should leave it like this?